src/components/ui/Button/buttons.scss

```diff
@@ -63,4 +66,39 @@
       &:active { background-color: var(--bg-lighten-pressed); }
     }
   }
+
+  &[data-button-variant="tertiary"] {
+    color: var(--color-link);
+
+    &:hover, &:focus { background-color: var(--bg-secondary-light); }
+
+    &:active {
+      color: var(--color-text-display);
+      background-color: var(--color-secondary-1);
+    }
+
+    &:disabled, &[data-button-disabled="true"] {
+      color: var(--color-text-disabled);
+      background-color: var(--bg-disabled);
+    }
+
+    &[data-button-inverse] {
+      color: var(--color-text-display);
+      background-color: var(--bg-neutral-lightest);
+
+      &:hover, &:focus { background-color: var(--bg-secondary-light); }
+
+      &:active { background-color: var(--color-secondary-1); }
+    }
+  }
+}
+
+.link-button[data-store-button] {
+  &:hover {
+    text-decoration: none;
+  }
+
+  &:disabled, &[data-button-disabled="true"] {
```

Comment:
Could you leave the `pointer-events: none;` only for `[data-button-disabled="true"]`? That way the `cursor: not-allowed` will work for the `<button>` at least.